docs/minio-client-complete-guide.md

```diff
@@ -352,8 +352,9 @@ USAGE:
    mc cp [FLAGS] SOURCE [SOURCE...] TARGET
    
 FLAGS:
-  --help, -h                       Show help.
-  --recursive, -r		   Copy recursively.
+  --recursive, -r                    Copy recursively.
+  --storage-class value, --sc value  Set storage class for object.
```

Comment:
Should `--storage-class value, --sc value  Set storage class for object.` be `--storage-class value, -sc value  Set storage class for object.` ?